src/components/ContentNav/Item/index.tsx

```diff
@@ -10,9 +10,9 @@ import { SubItemProps } from '../SubItem';
  */
 export interface ItemProps extends SubItemProps {
   expanded?: boolean;
-  subItems?: SubItemProps[];
+  subItems: SubItemProps[];
   expand?: (e: React.MouseEvent<SVGElement>, n: string) => void;
-  children?: JSX.Element[] | JSX.Element;
+  children?: JSX.Element[] | JSX.Element | boolean;
```

Comment:
I don't think we need to explicitly define children here, React passes the `children` prop by default along with other defined props and it always contains a type `JSX.Element` of a possible bool if there are no children.

Reply:
I am unsure on this one. I will need some advice one how to change this.